src/xcode/ENA/ENA/Source/Developer Menu/Features/DMCCLConfiguration/DMCCLConfigurationViewModel.swift

```diff
@@ -0,0 +1,76 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+#if !RELEASE
+
+import Foundation
+import UIKit
+import CertLogic
+
+final class DMCCLConfigurationViewModel {
+
+	// MARK: - Init
+
+	init(
+		restServiceProvider: RestServiceProviding
+	) {
+		self.restService = restServiceProvider
```

Comment:
I think the rest service is not needed anymore since we only set a flag now

Reply:
Removed.